superset-frontend/src/views/CRUD/hooks.ts

```diff
@@ -448,7 +448,10 @@ export function useImportResource(
                 t(
                   'An error occurred while importing %s: %s',
                   resourceLabel,
-                  error.errors.map(payload => payload.message).join('\n'),
+                  [
+                    ...error.errors.map(payload => payload.message),
+                    'Please re-export your file and try importing again',
```

Comment:
was this required/checked with the team?
Also, this should be translatable

Reply:
Thanks for the catch! A few thoughts:
* Are we sure that re-export and trying again will reliably fix the issue? I just want to make sure if we tell the user to take an action on the error that they have all the information they need to succeed at it.
* Definitely do wrap UI strings in `t()` as a general practice
* @kasiazjc has been capturing and evaluating error messages like this, and may have input on this one as well (cc @yousoph as well).

Reply:
> Thanks for the catch! A few thoughts:
> 
> * Are we sure that re-export and trying again will reliably fix the issue? I just want to make sure if we tell the user to take an action on the error that they have all the information they need to succeed at it.
> * Definitely do wrap UI strings in `t()` as a general practice
> * @kasiazjc has been capturing and evaluating error messages like this, and may have input on this one as well (cc @yousoph as well).

I did the repro steps (including removing datasets folder, changing folder) and for some weird reason it worked for me and there was no hanging/error 🥲 so I am not sure what is the flow now. 

In the screenshot the error is shown at the bottom right, which is not intuitive for the user as they are focused on the popover/modal and there is no visual relationship between the two if that makes sense. I think the error should be shown as an alert in the popover, but I would need more context.

Reply:
> was this required/checked with the team? Also, this should be translatable

This text was checked with the team. Please check the ticket. (38840)

Reply:
@kasiazjc , currently we are showing all db connection error using toast at the bottom-right corner. 
Do we need to change this?

Reply:
> @kasiazjc , currently we are showing all db connection error using toast at the bottom-right corner. Do we need to change this? @evans ^^

Agree that the bottom right toast is not the best place for error messages, but I think we can address that in a later effort instead of trying to do it in this fix. 
(Also I think the tag is mean for @rusackas 😄 )